dask/array/core.py

```diff
@@ -2439,7 +2439,8 @@ def map_overlap(self, func, depth, boundary=None, trim=True, **kwargs):
         >>> import dask.array as da
         >>> x = np.arange(16).reshape((4, 4))
         >>> d = da.from_array(x, chunks=(2, 2))
-        >>> d.map_overlap(lambda x: x + x.size, depth=1).compute()
+        >>> y = d.map_overlap(lambda x: x + x.size, depth=1, boundary='reflect')
```

Comment:
I've updated the doctests so they don't fail when the warning is raised.
But perhaps it would be better to specify `boundary=`none'` and to update the result values accordingly? That's the thing we want to encourage, really.

Reply:
On second thoughts, no. Looking more closely at these examples, I don't think that makes sense to do here.